sources/packages/backend/apps/api/src/route-controllers/application-exception/_tests_/application-exception-helper.ts

```diff
@@ -0,0 +1,59 @@
+import { TestingModule } from "@nestjs/testing";
+import { SystemUsersService } from "@sims/services";
+import {
+  Application,
+  ApplicationException,
+  ApplicationExceptionStatus,
+  Student,
+  User,
+} from "@sims/sims-db";
+import {
+  createFakeApplication,
+  createFakeApplicationException,
+  createFakeStudent,
+} from "@sims/test-utils";
+import { DataSource, Repository } from "typeorm";
+
+/**
+ * Create a fake application with an application exception associated.
+ * @param applicationExceptionStatus application exception status.
+ * @param dataSource application dataSource.
+ * @param module application module.
+ * @returns application with an application exception associated.
+ */
+export async function createFakeApplicationWithApplicationException(
```

Comment:
The idea so far would be that methods that are actually saving fake data would be named `saveFakeSomething`.